docs/protocol.rst

```diff
@@ -1,5 +1,5 @@
 ZEO Network Protocol (sans authentication)
-==========================================
+------------------------------------------
```

Comment:
If changing adornments, please make them more standard, not less standard.

Reply:
Done.